source/digital_wallets/index.html.md.erb

```diff
@@ -49,6 +65,71 @@ You get a Google Pay merchant ID from your Worldpay merchant admin interface.
 
 1. Go to the **Google Pay** setting, select **On** and enter the merchant ID.
 
+### Disable Google Pay
+
+1. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk).
+1. Under **Services**, select the service where you want to disable Google Pay.
+1. Select **Settings**.
+1. Change **Google Pay** to **Off**.
+
+## Test digital wallet payments
+
+You can test digital wallet payments using a GOV.UK Pay test account.
+
+### Test Google Pay
+
+To test Google Pay, you need a Google account and you must add a credit or debit card to that account. You can add a real credit or debit card, or [use Google Pay’s test cards](https://developers.google.com/pay/api/android/guides/resources/test-card-suite).
+
+We will not take any money if you use a real credit or debit card.
+
+To test Google Pay:
+
+1. In your test service, [create a payment through the API](https://docs.payments.service.gov.uk/making_payments/#creating-a-payment) or create a payment link through the GOV.UK Pay admin tool.
+
+1. Open Chrome.
+
+1. Sign in to a Google account.
+
+1. In your chosen browser, visit the `next_url` you received when you created the test payment or visit the payment link.
+
+1. Complete the test payment using a real card or a test card. We will not take any money if you use a real debit or credit card.
+
+You can see the payment in the GOV.UK Pay admin tool and through the API.
+
+### Test Apple Pay
+
+To test Apple Pay, you must set up Apple Pay on an iOS or macOS device. You can use a real credit or debit card. 
+
+We will not take any money if you add a real credit debit card to test Apple Pay. 
+
+If your organisation has an Apple developer account, you may be able to use [Apple’s test cards](https://developer.apple.com/apple-pay/sandbox-testing/).
+
+To test Apple Pay:
+
+1. In your test service, [create a payment through the API](https://docs.payments.service.gov.uk/making_payments/#creating-a-payment) or create a payment link through the GOV.UK Pay admin tool.
+
+1. On an iOS device, open a browser that supports Apple Pay, such as Safari, Chrome, or Microsoft Edge. If you’re using a Mac, open Safari.
+
+1. In your chosen browser, visit the `next_url` you received when you created the test payment or visit the payment link.
+
+1. Complete the test payment using the credit or debit card you added to your Apple Wallet. We will not take any money.
+
+### Test failed digital wallet payments
```

Comment:
When Iast talked to Sacha, we were going to leave out the magic values to keep things simple for now. Has there been a decision that it's now worthwhile adding them?

Reply:
Ah, I didn't know that was the case. Nat has added them (`DECLINED` and `3DS`) so I documented them.

Is it worth keeping the `DECLINED` one just for testing unhappy paths? Or do you think that'll confuse things?